helios-services/src/main/java/com/spotify/helios/agent/AgentConfig.java

```diff
@@ -369,4 +371,13 @@ public AgentConfig setConnectionPoolSize(final int connectionPoolSize) {
     this.connectionPoolSize = connectionPoolSize;
     return this;
   }
+
+  public File getGoogleCloudAccountCredentials() {
```

Comment:
Derp.

That seems less than ideal. For example, if I have a build agent in GCP that both pulls and pushes images to GCR using something like `dockerfile-maven-plguin`. Then I can't use the SA that is associated with the VM? Instead I must create a SA key and add glue to make sure it's loaded and used?